MigrationV3V4/CSharp/Skills/V3PizzaBot/PizzaOrderDialog.cs

```diff
@@ -85,7 +85,7 @@ private async Task PizzaFormComplete(IDialogContext context, IAwaitable<PizzaOrd
                 await context.PostAsync("You canceled the form!");
 
                 // If the user cancels the skill, send an `endOfConversation` activity to the skill consumer.
-                await SkillsHelper.EndSkillConversation(context.Activity as Activity);
+                await ConversationHelper.EndConversation(context.Activity as Activity, endOfConversationCode: EndOfConversationCodes.UserCancelled);
```

Comment:
Shouldn't you also call clear conversation state after this?

Reply:
The Form itself has been cancelled.